here-naksha-app-service/src/test/java/com/here/naksha/app/service/StorageApiTest.java

```diff
@@ -138,6 +141,32 @@ void tc0021_testGetStorageByWrongId() throws Exception {
     assertEquals(streamId, getHeader(response, HDR_STREAM_ID), "StreamId mismatch");
   }
 
+  @Test
+  void tc0022_testGetStorageNoPasswords() throws Exception {
+    // Test API : GET /hub/storages/{storageId}
+    // 1. Load test data, create storage
+    final String bodyJson = loadFileOrFail("StorageApi/TC0022_getStorageNoPasswords/create_storage.json");
+    final String streamId = UUID.randomUUID().toString();
+    HttpResponse<String> response = getNakshaClient().post("hub/storages", bodyJson, streamId);
+    assertEquals(200, response.statusCode(), "ResCode mismatch");
+    final JsonNode postResponse = new ObjectMapper().readTree(response.body());
```

Comment:
line 152 to 156 is repeating in this class .. better it is standardized as reusable method